### ISSUE TITLE
ATProto => AP: for quotes of a bridged post, include the original Fedi URL in "RE"

### ISSUE THREAD

**Issue by mackuba** [snarfed/bridgy-fed#1145]
Example:

This is a view of my post: https://bsky.app/profile/mackuba.eu/post/3kved2m6p4c2d - which is quoting Laurens' bridged post - as seen on Misskey. The bridged Misskey view of my post correctly renders the quote of the original post by Laurens, which is here: https://indieweb.social/@laurenshof/112327889750731757.

However, the fallback link in the text body of my bridged post links to the Bluesky-bridged version of Laurens' post, i.e.: `RE: https://bsky.app/profile/did:plc:43qcmg2xbxakj6mp4puqyayv/post/3kqvnekfgcwc2`. It would be nicer if it instead detected that this is a bridged post and linked instead to the original Fedi post, i.e. `RE: https://indieweb.social/@laurenshof/112327889750731757`.

![bafkreiaageralrbz3jcgizwwo3osg4zpe33ativov47y5bf7t3esnfrmbi](https://github.com/snarfed/bridgy-fed/assets/28465/0ff836e6-9244-4c5f-a30d-bc47be78535e)

**Comment by snarfed** [comment]
Yes! Great idea. I suspect this will be nontrivial, not high priority, but still worth tracking.
